main.go

```diff
@@ -80,13 +80,13 @@ func main() {
 		if len(os.Args) < 2 {
 			flag.CommandLine.Usage()
 		}
-		os.Exit(-1)
+		os.Exit(1)
```

Comment:
Can you use a variable named like general_error_code or add a commens so that other people can be aware the conversion?

Reply:
@northtyphoon + @shhsu - done. Also refactored the other const files to be one-liners rather than duplicating `const` everywhere.